src/Plugin/EntityBrowser/Widget/View.php

```diff
@@ -209,4 +210,41 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
     $this->selectEntities($entities, $form_state);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $options = [];
+    // Get only those enabled Views that have entity_browser displays.
+    $displays = Views::getApplicableViews('entity_browser_display');
+    foreach ($displays as $display) {
+      list($view_id, $display_id) = $display;
+      $view = $this->entityManager->getStorage('view')->load($view_id);
+      $options[$view_id . '.' . $display_id] = $this->t('@view : @display', array('@view' => $view->label(), '@display' => $view->get('display')[$display_id]['display_title']));
+    }
+
+    $form['view'] = [
+      '#type' => 'select',
+      '#title' => $this->t('View : View display'),
+      '#default_value' => $this->configuration['view'] . '.' . $this->configuration['view_display'],
+      '#options' => $options,
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitConfigurationForm(array &$form, FormStateInterface $form_state) {
```

Comment:
I think this is such a case. The sitebuilder has to select a single view display for the widget, but this is stored as a (view, view_display) pair. I think it's easier to use a single dropdown for this (as the UI), but I didn't want to change how this config is stored (so I didn't have to change even more things). That's why I added a submitConfigurationForm method that is specific to this class: it stores one single input as two different strings.

(I was thinking about two dropdowns first: one for selecting the view, the other selecting a display of that view – but this approach would yield even more changes like an AJAX call since the display dropdown's content depends on the current selection in the view dropdown, etc. So I've taken the easier road here.)